src/CivicrmEntityViewsData.php

```diff
@@ -127,6 +127,8 @@ public function getViewsData() {
               'entity_type' => $this->entityType->id(),
               'base field' => $this->entityType->getKey('id'),
               'field_name' => $field_name,
+              'field table' => $target_base_table,
+              'field field' => $field_name,
```

Comment:
The tests were failing, and other places in Drupal core have these set.

Reply:
I tried testing a View listing Events, adding relationship to Location Block, then another from Location Block to Address. 
I feel like the location block is one of these auto-generated relationships created by this code here. 
Not seeing any problems.